DOCKERFILE

```diff
@@ -0,0 +1,15 @@
+FROM tiangolo/uvicorn-gunicorn-fastapi:python3.8-slim as compile-image
+
+WORKDIR /
+RUN apt-get update && \
+    apt-get -y install gcc mono-mcs && \
+    rm -rf /var/lib/apt/lists/*
+COPY ./requirements.txt /requirements.txt
+RUN pip install --no-cache-dir --user -r requirements.txt; rm -r /tmp 
+
+FROM tiangolo/uvicorn-gunicorn-fastapi:python3.8-slim
+COPY --from=compile-image /root/.local /root/.local
+COPY ./app /app
+COPY ./crawler /app/crawler
```

Comment:
This might not work as in the original project and the development dockerfile crawler is supposed to be outside the app folder